File: PathoFroh/src/main/java/com/patho/main/repository/jpa/MaterialPresetRepository.java
package com.patho.main.repository.jpa;

import com.patho.main.model.MaterialPreset;
import com.patho.main.repository.jpa.custom.MaterialPresetRepositoryCustom;

public interface MaterialPresetRepository
        extends BaseRepository<MaterialPreset, Long>, MaterialPresetRepositoryCustom {

}

File: PathoFroh/src/main/kotlin/com/patho/main/repository/jpa/MaterialPresetRepository.kt
package com.patho.main.repository.jpa

import com.patho.main.model.preset.MaterialPreset
import com.patho.main.model.preset.StainingPrototype
import com.patho.main.repository.jpa.custom.MaterialPresetRepositoryCustom

interface MaterialPresetRepository : MaterialPresetRepositoryCustom, BaseRepository<MaterialPreset, Long> {
}